subject: "Rescue keyword"
description: "exceptions list / without explicit exceptions list"
notes: >
  The rescue branch is represented as a RescueStandardErrorNode node.
focused_on_node: "org.truffleruby.language.exceptions.TryNode"
ruby: |
  begin
    "foo"
  rescue
    42
  end
ast: |
  TryNodeGen
      attributes:
          canOmitBacktrace = true
      children:
          rescueParts = [
              RescueStandardErrorNode
                  children:
                      rescueBody =
                          IntegerFixnumLiteralNode
                              attributes:
                                  value = 42
          ]
          tryPart =
              StringLiteralNode
                  attributes:
                      encoding = UTF-8
                      tstring = foo